master/jobmaster/benchmark/master.go

```diff
@@ -226,10 +256,23 @@ func (m *Master) monitorSchedulingTasks() {
 			// cancel it
 			//}
 
-			log.L().Logger.Info("begin to reschedule task group")
+			log.L().Logger.Info("begin to reschedule task group", zap.Any("group", group))
 			if err := m.reScheduleTask(group); err != nil {
 				log.L().Logger.Error("cant reschedule task", zap.Error(err))
-				// FIXME: this will cause deadlock problem
```

Comment:
I think this problem still remain when we read and write the same channel `scheduleWaitingTasks` and this channel is full. So leave this comments and we should address it in the furture

Reply:
but after we schedule "a group of tasks", this problem is mitigated

Reply:
Oh yes, the channel could be full, I misunderstand the meaning.